.env.test

```diff
@@ -10,4 +10,4 @@ EMAIL_FROM="LabelFlow <contact@labelflow.ai>"
 LABELFLOW_AWS_ACCESS_KEY_ID=minioadmin
 LABELFLOW_AWS_SECRET_ACCESS_KEY=minioadmin
 LABELFLOW_AWS_REGION=eu-west-1
-LABELFLOW_AWS_ENDPOINT=localhost:9000
\ No newline at end of file
+LABELFLOW_AWS_ENDPOINT=http://localhost:9000
```

Comment:
Don't we need here as well all the other env vars you added in `.env.development`?
```
MINIO_ROOT_USER=minioadmin
MINIO_ROOT_PASSWORD=minioadmin
LABELFLOW_AWS_ACCESS_KEY_ID=minioadmin
LABELFLOW_AWS_SECRET_ACCESS_KEY=minioadmin
LABELFLOW_AWS_REGION=eu-west-1
LABELFLOW_AWS_ENDPOINT=http://localhost:9000 
```

Reply:
Hum good question
Classic "on my machine, it works" but to be honest, now that you say it, I don't understand why it works ahah.
I'll investigate.

I know that the correct env vars are set on the docker-compose file, but, still it should ask need the credentials while testing

Reply:
Are they set in your `env.local` maybe?

Reply:
Ah no, I found out why. I didn't do the change in this PR, because they were already on the `.env.test` file (look at the lines above your comments). 
Only `MINIO_ROOT_USER=minioadmin` & `MINIO_ROOT_PASSWORD=minioadmin` are missing, but they are only needed if you want to start a minio server (so on the docker-compose). To dialog with the server, we only need the AWS credentials as we use the AWS SDK.

Reply:
Ah yes it makes sense! Thanks for the explanation